syft/workers/node_client.py

```diff
@@ -248,10 +250,31 @@ def serve_model(
             "allow_download": str(allow_download),
             "mpc": str(mpc),
             "allow_remote_inference": str(allow_remote_inference),
-            "model": serialized_model.decode(self.encoding),
+            "model": serialized_model,
         }
-        response = self._forward_json_to_websocket_server_worker(message)
-        return self._return_bool_result(response)
+
+        url = self.address.replace("ws", "http") + "/serve-model/"
+
+        # Multipart encoding
+        form = MultipartEncoder(message)
+        upload_size = form.len
```

Comment:
Can `form.len` be `None` ? If so, can be put a check and maybe raise a error of message size being null.